packages/opencensus-web-core/test/test-url-util.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'jasmine';
+import {parseUrl} from '../src/common/url-util';
+
+describe('parseUrl', () => {
+  it('parses a URL into its various components', () => {
+    const parsedUrl =
```

Comment:
Can we add one more test without port? something like ```http://example.com/path1/path2?search=test&format=json```

Reply:
Sure, done.